hca/util/tsv.py

```diff
@@ -0,0 +1,8 @@
+import csv
+import functools
+
+
+# Wrap the csv library with our required options
+
+DictReader = functools.partial(csv.DictReader, delimiter='\t', dialect='excel-tab')
```

Comment:
Nifty, but I think for type checking, auto-completion and ensuring that callers don't accidentally pass `delimiter` and `dialect` (only to have those be silently ignored) it might be better to make this an explicit def whose signature has only the file name parameter. It's easier to start strict and relax the signature over time than the other way around. I'd be happy to have this be four lines instead of two if I get clarify